.circleci/config.yml

```diff
@@ -234,43 +232,161 @@ jobs:
 
-
-
+orbs:
+  codecov: codecov/codecov@1.0.5
+  azure-cli: circleci/azure-cli@1.0.0
 
 workflows:
   version: 2
+
+  performance_tests:
+    jobs:
+      - ch_benchmark:
+          filters:
+            branches:
+              only:
+                - /ch_benchmark\/.*/ # match with ch_benchmark/ prefix
+      - tpcc_benchmark:
+          filters:
+            branches:
+              only:
+                - /tpcc_benchmark\/.*/ # match with tpcc_benchmark/ prefix
+
   build_and_test:
     jobs:
-      - build
-      - check-style
-      - check-sql-snapshots
+      - build:
+          filters:
+            branches:
+              ignore:
+                - /tpcc_benchmark\/.*/ # ignore tpcc_benchmark/
+                - /ch_benchmark\/.*/ # ignore ch_benchmark/
+      - check-style:
+          filters:
+            branches:
+              ignore:
+                - /tpcc_benchmark\/.*/ # ignore tpcc_benchmark/
+                - /ch_benchmark\/.*/ # ignore ch_benchmark/
+      - check-sql-snapshots:
+          filters:
+            branches:
+              ignore:
+                - /tpcc_benchmark\/.*/ # ignore tpcc_benchmark/
+                - /ch_benchmark\/.*/ # ignore ch_benchmark/
 
       - test-11_check-multi:
           requires: [build]
+          filters:
+            branches:
+              ignore:
+                - /tpcc_benchmark\/.*/ # ignore tpcc_benchmark/
+                - /ch_benchmark\/.*/ # ignore ch_benchmark/
       - test-11_check-tt-van-mx:
           requires: [build]
+          filters:
+            branches:
+              ignore:
+                - /tpcc_benchmark\/.*/ # ignore tpcc_benchmark/
+                - /ch_benchmark\/.*/ # ignore ch_benchmark/
       - test-11_check-iso-work-fol:
           requires: [build]
+          filters:
+            branches:
+              ignore:
+                - /tpcc_benchmark\/.*/ # ignore tpcc_benchmark/
+                - /ch_benchmark\/.*/ # ignore ch_benchmark/
       - test-11_check-fol:
           requires: [build]
+          filters:
+            branches:
+              ignore:
+                - /tpcc_benchmark\/.*/ # ignore tpcc_benchmark/
+                - /ch_benchmark\/.*/ # ignore ch_benchmark/
       - test-11_check-failure:
           requires: [build]
+          filters:
+            branches:
+              ignore:
+                - /tpcc_benchmark\/.*/ # ignore tpcc_benchmark/
+                - /ch_benchmark\/.*/ # ignore ch_benchmark/
 
       - test-12_check-multi:
           requires: [build]
+          filters:
+            branches:
+              ignore:
+                - /tpcc_benchmark\/.*/ # ignore tpcc_benchmark/
+                - /ch_benchmark\/.*/ # ignore ch_benchmark/
       - test-12_check-tt-van-mx:
           requires: [build]
+          filters:
+            branches:
+              ignore:
+                - /tpcc_benchmark\/.*/ # ignore tpcc_benchmark/
+                - /ch_benchmark\/.*/ # ignore ch_benchmark/
```

Comment:
I suppose if the prefix is benchmark_ then we only need a single line here

Reply:
I have changed the regex to be single line : `/(tpcc|ch)_benchmark\/.*/`

Were you suggesting that we should have `benchmark_tpcc/` or and `benchmark_ch/` and then we can just check `benchmark_` prefix?

Reply:
> Were you suggesting that we should have `benchmark_tpcc/` or and `benchmark_ch/` and then we can just check `benchmark_` prefix?

Yes, I think what Marco suggest is `/benchmark_.*/`